sbws/util/config.py

```diff
@@ -29,52 +39,57 @@ def _read_config_file(conf, fname):
     return conf
 
 
+def _extend_config(conf, fpath):
+    """Extend ConfigParser from file configuration."""
+    conf = _read_config_file(conf, fpath)
+    return conf
+
```

Comment:
Unless I'm missing something, `_extend_config()` is an unnecessary wrapper around `_read_config_file()`, thus every time `_extend_config()` is called, `_read_config_file()` could be called instead.

If you like the name `_extend_config()` better (I do), I'd just rename `_read_config_file()` to that.